docs/getting-started.md

```diff
@@ -17,22 +17,46 @@ instance using Ubuntu 18.04 on EC2. Firecracker uses
 [KVM](https://www.linux-kvm.org) and needs read/write access that can be
 granted as shown below:
 
-```
-sudo setfacl -m u:${USER}:rw /dev/kvm
-```
+ ```
+ sudo setfacl -m u:${USER}:rw /dev/kvm
+ ```
 
 The generic requirements are explained below:
 
 - **Linux 4.14+**
 
   Firecracker currently supports physical Linux x86_64 hosts, running kernel
-  version 4.14 or later.
+  version 4.14 or later. Please check first if your Linux host supports vitualization
+  running as root the following command:
+  
+  ```
+  egrep '(vmx|svm)' --color=always /proc/cpuinfo
+  ```
+  you should see the following answer (number of flags depends of the number of cores on the CPU):
+  
+  ```
+  flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx   mmxext fxsr_opt pdpe1gb rdtscp lm 3dnowext 3dnow constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid pni monitor cx16   popcnt lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt nodeid_msr         hw_pstate vmmcall npt lbrv svm_lock nrip_save
+  flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx   mmxext fxsr_opt pdpe1gb rdtscp lm 3dnowext 3dnow constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid pni monitor cx16   popcnt lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt nodeid_msr         hw_pstate vmmcall npt lbrv svm_lock nrip_save
+
+  ```
+  if no answer appears, then virtualization is not supported by your CPU.  
 
 - **KVM**
 
-  Please make sure that:
-  1. you have KVM enabled in your Linux kernel, and
-  2. you have read/write access to `/dev/kvm`.
+  Please make sure that you have KVM enabled in your Linux kernel, to do so
+  you can run:
+  
+  ```
+  lsmod | grep '^kvm'
+  ```
+  you should see an answer like the following:
+  
+  ```
+  kvm-intel             86016  0
+  kvm                   598016  1 kvm-intel
+  ```   
+  
+  You also need to have read/write access to `/dev/kvm`.
```

Comment:
I'm curious as to why there are leading whitespaces on some lines and not on the others - until we figure that out, please adhere to the existing style and in this case add some leading spaces to align with the lines below.

Reply:
@aghecenco thank you for your suggestions. All corrections done.